kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiBatchSessionImpl.scala

```diff
@@ -77,6 +77,8 @@ class KyuubiBatchSessionImpl(
       sessionManager.validateBatchConf(batchRequest.getConf.asScala.toMap)
   }
 
+  override lazy val name: Option[String] = Option(batchRequest.getName)
```

Comment:
I think it is not needed to respect `KyuubiConf.SESSION_NAME.key` here
@ulysses-you

Reply:
please ignore